lib/tasks/send_review_reminders.rake

```diff
@@ -0,0 +1,11 @@
+desc "Send review reminder emails for when the review_at datetime has passed"
+task send_review_reminders: :environment do
+  ReviewReminder
+  .joins(document: :latest_edition)
+  .where(reminder_sent_at: nil)
+  .where("review_at < ?", Time.zone.today)
+  .where.not(document: { editions: { first_published_at: nil } })
```

Comment:
I'm not sure if this is overkill but it didn't really seem to make much sense to send a review email for something that's never been published.